rust/gui-client/src-common/src/logging.rs

```diff
@@ -111,6 +112,7 @@ pub async fn export_logs_to(path: PathBuf, stem: PathBuf) -> Result<()> {
     })
     .await
     .context("Failed to join zip export task")??;
+    tracing::debug!(elapsed_s = start.elapsed().as_secs(), "Exported logs");
```

Comment:
about 7 seconds in debug, about 0 in release, had to check

Reply:
`Duration` has a nicely formatted debug output actually.